specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2019-11-01/costmanagement.json

```diff
@@ -1453,6 +1531,16 @@
           "format": "date-time",
           "readOnly": true
         },
+        "dateRange": {
```

Comment:
Any changes to existing contracts must go into a new API version.  ViewProperties are used by existing APIs in 2019-11-01

Reply:
It looks like a change in the contract, but this property is already part of the contract but was somehow missed in the previews arm review. I am adding it to fix some swagger correctness items in S360. 
PS: Currently, Cost Management UX (the team I am in) is the only consumer of this api.

Reply:
If this has been in the public API but was only missed in swagger then I don't have any API review concerns but you'll have to work with the SDK reviewer (@ArcturusZhang ) to ensure it doesn't impact SDKs